### PR TITLE
BAEL-20858 move spring boot camel

### PR DESCRIPTION
**Pull request by Maiklins**

### CODE REVIEW

File: spring-boot-modules/spring-boot-camel/pom.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
         <groupId>com.baeldung</groupId>
         <artifactId>parent-modules</artifactId>
         <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../parent-modules</relativePath>
```
Comment (dkapil) [inline spring-boot-modules/spring-boot-camel/pom.xml]: if the parent is parent-modules, then we can easily switch it to parent-modules as well, let's do that.

Comment (Maiklins) [reply spring-boot-modules/spring-boot-camel/pom.xml]: do you mean to change it to parent-boot-2?

Comment (dkapil) [reply spring-boot-modules/spring-boot-camel/pom.xml]: @Maiklins sorry, I meant we can change the parent to the spring-boot-submodules i.e. the actual parent

Comment (Maiklins) [reply spring-boot-modules/spring-boot-camel/pom.xml]: Done

**Pull request merged**